styles/styles.css

```diff
@@ -20,44 +20,43 @@
   --text-color: black;
 
   /* fonts */
-  --body-font-family: roboto, roboto-fallback;
+  --body-font-family: avenir-next;
   --heading-font-family: var(--body-font-family);
-  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
 
-  /* body sizes */
-  --body-font-size-m: 22px;
-  --body-font-size-s: 18px;
-  --body-font-size-xs: 16px;
-
-  /* heading sizes */
-  --heading-font-size-xxl: 48px;
```

Comment:
These variables are important because they're used here: https://github.com/aemsites/esri/blob/d54f2255cab85b4168080f9fbe5bc2ad5caea1d7/styles/styles.css#L93

What do all these "font--<number>" mean?

Reply:
Yes, we have a variable font scale set up based on that scale in `:root`